modulefiles/jet.intel/fv3_debug

```diff
@@ -0,0 +1,42 @@
+#%Module
+
+proc ModulesHelp {} {
+  puts stderr "\tcit - loads modules required for building and running UFS Model on Jet/Intel"
+}
+
+module-whatis "loads UFS Model prerequisites for Jet/Intel"
+
+module use /contrib/sutils/modulefiles
+module load sutils
+
+module load cmake/3.16.1
+setenv CMAKE_C_COMPILER mpiicc
+setenv CMAKE_CXX_COMPILER mpiicpc
+setenv CMAKE_Fortran_COMPILER mpiifort
+setenv CMAKE_Platform jet.intel
+
+module use /lfs4/HFIP/hfv3gfs/nwprod/hpc-stack/libs/modulefiles/stack
+
+module load hpc/1.1.0
+module load hpc-intel/18.0.5.274
+module load hpc-impi/2018.4.274
+
+module load jasper/2.0.22
+module load zlib/1.2.11
+module load png/1.6.35
+
+module load hdf5/1.10.6
+module load netcdf/4.7.4
+module load pio/2.5.1
+module load esmf/8_1_0_beta_snapshot_27
```

Comment:
Is there no esmf debug module on jet ?

Reply:
For all the platforms where we haven't compiled the debug module yet we simply had to copy the existing module over so that the code would still run. This is because somebody removed the logic (or it is not working as intended) that says "only if a debug module exists, use it; otherwise use the standard module".

Yes, when the next HPC stack release is rolled out we should create debug modules for all platforms.

Reply:
FYI, it's the same for wcoss_cray, for example.